packages/@okta/vuepress-theme-prose/components/Sidebar.vue

```diff
@@ -1,14 +1,14 @@
 <template>
   <aside class="landing-navigation" :class="{active: sidebarActive}">
     <ul class="landing">
-      <SidebarItem v-for="(link, key) in navigation" :key="key" :link="link" />
+      <SidebarItem v-for="link in navigation" :key="link.title" :link="link" />
```

Comment:
What was the purpose of changing the `:key` item here?

Reply:
@bretterer mentioned that in PR description:
>when we have non-flat arrays using indexes is not a best solution - several elements will have same "key" properties, this can cause side effects when components are updated and re-rendered. 

In detail - Vue optimizes DOM operations using key properties(basically, as any other framework, such as Angular or React). When Vue see several elements with same keys it will not be happy about that, will get confused and will swap whole DOM representation, instead of particular element with particular key ID. 

There is a link to docs, it's not super in-depth, but explains better than me
https://vuejs.org/v2/guide/list.html#Maintaining-State